core/src/components/button/button.ios.vars.scss

```diff
@@ -16,7 +16,7 @@ $button-ios-margin-bottom:                            4px !default;
 $button-ios-margin-start:                             2px !default;
 
 /// @prop - Padding top of the button
-$button-ios-padding-top:                              0 !default;
+$button-ios-padding-top:                              13px !default;
```

Comment:
Where did all the new padding values come from?

Reply:
Good question! These values are the maximum amount that could be added to ensure that wrapped text will have some padding above/below the text without increasing the height of the default (non-wrapped) button. 

Here are some examples with and without padding:

| no padding | added padding |
|---| ---|
| ![brandy ngrok app_src_components_button_test_wrap(iPhone 12 Pro) (1)](https://github.com/ionic-team/ionic-framework/assets/6577830/c7d42083-edef-498d-894a-bc0971b04829) | ![brandy ngrok app_src_components_button_test_wrap(iPhone 12 Pro)](https://github.com/ionic-team/ionic-framework/assets/6577830/48c7949d-c709-4da5-9878-9c76b7f93f7c) |